tikv-client/src/test/java/com/pingcap/tikv/txn/LockResolverTest.java

```diff
@@ -84,7 +84,7 @@ private boolean prewrite(List<Mutation> mutations, long startTS, Mutation primar
     for (Mutation m : mutations) {
       Pair<TiRegion, Store> pair = session.getRegionManager().getRegionStorePairByKey(m.getKey());
 
-      RegionStoreClient client = RegionStoreClient.create(pair.first, pair.second, session);
+      RegionStoreClient client = builder.build(pair.first);
```

Comment:
If you intend to use builder.build(), than the pair is not needed since you only need to fetch TiRegion. i.e., via `session.getRegionManager().getRegionByKey(m.getKey());`

Reply:
fix